units/user-configdrive.service

```
@@ -21,6 +21,7 @@ ConditionKernelCommandLine=!coreos.oem.id=digitalocean
 
```

Should we still add
```suggestion
ConditionKernelCommandLine=!coreos.oem.id=openstack
ConditionKernelCommandLine=!flatcar.oem.id=openstack
```
to not have cloud-init run twice? Or is this invocation below different from `coreos-cloudinit --oem=ec2-compat` because it does not depend on having a metadata endpoint?

---

On a first look the code seems to be using both the config drive and the endpoint when `--oem=ec2-compat` is given.

---

that's right. It tries both.